src/main/resources/application.properties

```diff
@@ -1,8 +1,6 @@
-  spring.application.name=collab
+  spring.application.name=taskmanager
```

Comment:
DO NOT SEND CREDENTIALS TO GITHUB OR ANY REPOSITORY
Use environment variable

Reply:
Fixed in #6